tests/StacktraceTest.php

```diff
@@ -84,21 +88,20 @@ public function testExceptionHandlerStacktrace()
     public function testAnonymousFunctionStackframes()
     {
         $fixture = $this->getJsonFixture('backtraces/anonymous_frame.json');
-        $stacktrace = Bugsnag_Stacktrace::fromBacktrace($this->config, $fixture['backtrace'], 'somefile.php', 123)->toArray();
+        $stacktrace = Stacktrace::fromBacktrace($this->config, $fixture['backtrace'], 'somefile.php', 123)->toArray();
 
-        $this->assertCount(5, $stacktrace);
+        $this->assertCount(4, $stacktrace);
```

Comment:
There should be a line for the main method. Adding a few more tests to shake out what's happening.